unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GotoPanelHUD/Tests/GotoPanelControllerShould.cs

```diff
@@ -0,0 +1,29 @@
+using DCL;
+using NSubstitute;
+using NSubstitute.Extensions;
+using NUnit.Framework;
+using GotoPanel;
+
+namespace Test.GotoPanel
+{
+    public class GotoPanelControllerShould
+    {
+        private GotoPanelHUDController hudController;
+        private IGotoPanelHUDView hudView;
+
+        [SetUp]
+        public void SetUp()
+        {
+            hudView = Substitute.For<IGotoPanelHUDView>();
+            hudController = Substitute.ForPartsOf<GotoPanelHUDController>();
+            hudController.Configure().CreateView().Returns(info => hudView);
+            hudController.Initialize();
+        }
+
+        [Test]
+        public void Initialize()
+        {
+            Assert.AreEqual(hudView, hudController.view);
```

Comment:
i dont really see the purpose of testing that a view was created when it was mocked that way in the first place.. what exactly are you trying to validate?

Reply:
Trying to validate that the controller has the correct view (even if mocked)